jwst/lib/set_telescope_pointing.py

```diff
@@ -204,6 +209,37 @@ def add_wcs(filename, default_pa_v3=0., siaf_path=None, engdb_url=None,
     model.close()
     logger.info('...update completed')
 
+def update_mt_kwds(model):
+    """Add/update the Moving target header keywords
+
+    If the target type is "moving_target" check for the moving target position
+    table. If this is available calculate the moving target position keywords
+    and insert or update MT_RA & MT_DEC.
+    """
+
+    if model.extra_fits.hasattr('MOVING_TARGET_POSITION'):
```

Comment:
We should make access to this table not go through `extra_fits`.  Can we add this as an allowable table in the schema?

Reply:
Agree, we should add the table definition to the level-1b schema and then access it through the data model.